### PR TITLE
Upgrade dependencies to fix build

### PR DESCRIPTION
**Pull request by wuyiqicc**
Current build fails. After upgrading dependancies it works

### CODE REVIEW

File: decide/test_rust.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import rust
```
Comment (mattknox) [inline decide/test_rust.py]: where does it get this?  there must be something in the requirements.txt, no?

File: decide/test_rust.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+decider = rust.init("decision_maker_name", "cfg.json")
```
Comment (mattknox) [inline decide/test_rust.py]: we might want to make a minimal cfg.json rather than the one you included.

**Review by mattknox — CHANGES_REQUESTED** [review]
I think this is missing some requirements.txt?  Or does maturin install the relevant stuff?

File: decide/test_rust.py (hunk 2 above)
Comment (wuyiqicc) [reply decide/test_rust.py]: will do!

File: decide/test_rust.py (hunk 1 above)
Comment (wuyiqicc) [reply decide/test_rust.py]: i think `maturin develop` does the magic here. (https://github.com/PyO3/maturin)

Comment (wuyiqicc) [reply decide/test_rust.py]: it is defined in Cargo.toml: 
```
[lib]
name = "rust"
```

**Review by mattknox — CHANGES_REQUESTED** [review]
Only change I'd suggest here is that I don't see how this changes stuff for our downstream folks.  Like, if I currently have experiments.py in my requirements.txt, does this change anything (other than the version number, presumably)?

**Review by mattknox — APPROVED** [review]

**Pull request merged**